Templates/AppSource App/.AL-Go/localDevEnv.ps1

```diff
@@ -10,7 +10,8 @@ Param(
     [pscredential] $credential = $null,
     [string] $licenseFileUrl = "",
     [switch] $fromVSCode,
-    [switch] $accept_insiderEula
+    [switch] $accept_insiderEula,
+    [switch] $clean
```

Comment:
I think we should consider adding some docs to the localdevenv script. I don't think it's that obvious what "clean" entails in this context.

Reply:
Yeah - will add this after v5.1